package.json

```diff
@@ -33,22 +33,22 @@
     "node": ">= 6.0.0"
   },
   "devDependencies": {
-    "autobahn": "0.11.2",
-    "chai": "3.5.0",
-    "coveralls": "2.11.12",
-    "gulp": "3.9.1",
-    "gulp-clean": "0.3.2",
-    "gulp-jsbeautify": "0.1.1",
-    "gulp-sourcemaps": "2.0.0-alpha",
-    "gulp-typescript": "2.13.6",
-    "istanbul": "0.4.4",
-    "mocha": "2.5.3",
-    "run-sequence": "1.2.2",
-    "tslint": "3.13.0",
-    "typescript": "1.8.10"
+    "autobahn": "^0.11.2",
+    "chai": "^3.5.0",
+    "coveralls": "^2.11.12",
+    "gulp": "^3.9.1",
+    "gulp-clean": "^0.3.2",
+    "gulp-jsbeautify": "^0.1.1",
+    "gulp-sourcemaps": "^2.0.0-alpha",
+    "gulp-typescript": "^2.13.6",
+    "istanbul": "^0.4.4",
+    "mocha": "^2.5.3",
+    "run-sequence": "^1.2.2",
+    "tslint": "^3.13.0",
+    "typescript": "^1.8.10"
   },
   "dependencies": {
-    "debug": "2.2.0",
-    "ws": "4.0.0"
+    "debug": "^2.2.0",
+    "ws": "^4.0.0"
```

Comment:
I would prefer to not have it this way.. We already had similar issue with ws https://github.com/ivaylopivanov/wamp-server/pull/5

Reply:
I have enabled Greenkeeper to take care of the dependencies.